src/tictactoe/ai.clj

```diff
@@ -0,0 +1,49 @@
+(ns tictactoe.ai
+  (:require [tictactoe.board :refer [place-move winner game-over? available-spaces]]))
```

Comment:
You should basically never `refer`. It makes it harder to find where a function is coming from. I'd much prefer...

```
(ns tictactoe.ai
  (:require [tictactoe.board :as board]))
```

If you're in a large project, 300 lines deep in a namespace, and you want to see the implementation of some function that's being called, the having it have a namespace prefix like `board/place-move` makes it really obvious that "Hey, I need to go to another namespace to find that function."